stacks_queues/stack.py

```diff
@@ -12,7 +12,7 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_first(element)
 
     def pop(self):
```

Comment:
✨ This works, however you could also take advantage of the `remove_first` method in `LinkedList`